bin.src/validateDrp.py

```diff
@@ -69,6 +69,11 @@
             'matchRef': matchRef,
             }
 
+    if not args.configFile or not visitDataIds:
+        visitDataIds = util.discoverDataIds(args.repo)
+        print("VISITDATAIDS: ", visitDataIds)
+        kwargs = {}
+
```

Comment:
For these under-specified cases, is there any code to verify that the given filter exists for each visit?